cmd/cored/no_loopback_authn.go

```diff
@@ -0,0 +1,18 @@
+//+build no_loopback_authn
```

Comment:
Normally I am all on board with using "authn" or "authz" to distinguish. However, in this case @tessr will shortly be doing the authz project and as part of that I'm pretty sure this flag will change from modifying authn behavior to modifying authz behavior.

We could update the tag name at that time… or we could cheat and just call it "auth" now. :stare: